templates/cloud/index.html

```diff
@@ -0,0 +1,510 @@
+{% extends "cloud/base_cloud.html" %}
```

Comment:
this page doesn't exist any more right? redirects to `/openstack`. So I'm assuming this file shouldn't exist?